.azure/pipelines/jobs/default-build.yml

```diff
@@ -48,12 +46,11 @@
 
 parameters:
   agentOs: 'Windows'
-  poolName: ''
-  poolVmImage: ''
+  # pool: {} don't define an empty object default because there is no way in template expression yet to check "if isEmpty(parameters.pool)"
```

Comment:
Where is this parameter used? I suggest disabling overrides entirely unless absolutely necessary in one of the files in the parent directory.

Reply:
Unused. Will remove.